taskfiles/utils.yml

```diff
@@ -103,3 +103,35 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  # ===
+  # clang UTILS
+  # ===
+  clang-format:
+    internal: true
+    requires:
+      vars: ["G_LINT_VENV_DIR", "SRC_DIR", "FLAGS"]
```

Comment:
* G_LINT_VENV_DIR -> LINT_VENV_DIR
  * `G_` is only for [global variables](https://docs.yscope.com/dev-guide/contrib-guides-taskfiles.html#variable-naming).
* Let's put `FLAGS` before `SRC_DIR` to keep them [alphabetical](https://docs.yscope.com/dev-guide/contrib-guides-taskfiles.html#ordering-of-list-map-entries).